hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -94,8 +94,10 @@ public static void initialize(String clusterId, String scmId,
     try {
       server = newRaftServer(scmId, conf).setGroup(group).build();
       server.start();
-      // TODO: Timeout and sleep interval should be made configurable
-      waitForLeaderToBeReady(server, 60000, group);
+      final SCMHAConfiguration haConf = conf.getObject(SCMHAConfiguration.class);
+      long waitTimeout = haConf.getLeaderReadyWaitTimeout();
+      long retryInterval = haConf.getLeaderReadyCheckInterval();
+      waitForLeaderToBeReady(server, waitTimeout, retryInterval, group);
```

Comment:
How about just pass `conf` to `waitForLeaderToBeReady`, and read out `retryInterval` and `waitTimeout` there ?

Reply:
here, conf is an OzoneConfiguration object  which can not read out retryInterval and waitTimeout directly. So, for now , i have to get a SCMHAConfiguration to do this. It seems better to pass conf to waitForLeaderToBeReady,done.
maybe we can unify SCMHAConfiguration to OzoneConfiguration later, and then there will be only one Configuration for scm. what do you think about it?